Last-Update: 2012-12-25
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: correct png2theora executable name

--- a/cmd/seekwatcher
+++ b/cmd/seekwatcher
@@ -263,9 +263,9 @@
         figindex += 1
     a.hold(True)
 
     if mencoder == "png2theora":
-        os.system("png2theora -o %s %s" % (movie_name, fname) + '-%06d.png')
+        os.system("theora_png2theora -o %s %s" % (movie_name, fname) + '-%06d.png')
     else:
         os.system("mencoder mf://%s*.png -mf type=png:fps=%d -of mpeg -ovc lavc -lavcopts vcodec=mpeg2video:vbitrate=%s -oac copy -o %s" % (fname, movie_fps, options.movie_vbitrate, movie_name))
 
     for root, dirs, files in os.walk(png_dir):
@@ -662,9 +662,9 @@
 
 def check_for_mencoder(encoder_prog=all):
     # check for either supported media encoder when no arguments specified
     if encoder_prog == all:
-        r1 = check_for_mencoder("png2theora")
+        r1 = check_for_mencoder("theora_png2theora")
         if r1:
             return True
         return check_for_mencoder("mencoder")
 
@@ -898,9 +898,9 @@
             print "mencoder required for encoding mpegs. Try using -o fname.ogg"
             sys.exit(1)
         mencoder = "mencoder"
     elif options.output.endswith((".ogg", ".OGG")):
-        if check_for_mencoder("png2theora"):
+        if check_for_mencoder("theora_png2theora"):
             mencoder = "png2theora"
         else:
             # We can't get here unless mencoder is set to either mencoder or
             # png2theora, so if the encoder isn't the latter...
